tasks/README.md

```diff
@@ -33,12 +33,11 @@ Tasks are packaged and distributed in the `/tasks` directory of a Puppet module.
 - Filenames not matching the task name regular expression will be ignored.
 - Tasks are referred to as `module_name::task_name`.
 - Tasks must exist at the top level of the tasks directory to be found.
-- Task metadata is stored in `task_name.json` and is optional.
-- The executable for the task must be stored in `task_name` or `task_name.<extension>`(where extension != `json`, `md` or `conf`).
+- Task metadata is stored in `task_name.json`.
```

Comment:
Neither the ticket nor the commit messages contain a rationale for making task metadata mandatory, and it does not seem to be required for the rest of the additions here.

Reply:
:+1: